RockSniffer/RPC/DiscordRPCHandler.cs

```
@@ -70,6 +71,13 @@ internal void UpdatePresence()
             //If we have a valid song and are playing a song
             if ((songdetails != null && readout != null) && (state == SnifferState.SONG_STARTING || state == SnifferState.SONG_PLAYING || state == SnifferState.SONG_ENDING))
             {
+                // Get the appropriate album cover. In case of failure display returned text.
+                {
+                    (string? URL, string DisplayText) resultTuple = albumArtResolver.Get(songdetails);
```

Try/catch, fail silently / log error.